addons/nuxeo-diff/nuxeo-diff-content/src/main/resources/OSGI-INF/content-diff-config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.diff.settings">
+    <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+        <documentation>
+            Allows to limit the diff count between two files.
+            @since 10.10-HF64
+        </documentation>
+        <property name="nuxeo.diff.limit">10000</property>
+    </extension>
+</component>
```

Comment:
XML files have two spaces indentation.
```suggestion
<component name="org.nuxeo.ecm.diff.settings">
  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
    <documentation>
      Allows to limit the diff count between two files.
      @since 10.10-HF64
    </documentation>
    <property name="nuxeo.diff.limit">10000</property>
  </extension>
</component>
```